<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>color_filter_widget</class>
 <widget class="QWidget" name="color_filter_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Origin image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Image filter</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelOriginImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelImageFiltered">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxHSVFilter">
     <property name="title">
      <string>HSV filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Hue low</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSliderHueLow">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBoxHueLow">
        <property name="maximum">
         <number>179</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Hue up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSlider" name="horizontalSliderHueUp">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="value">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="spinBoxHueUp">
        <property name="maximum">
         <number>179</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Sat low</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="horizontalSliderSatLow">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBoxSatLow">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Sat up</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSlider" name="horizontalSliderSatUp">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="spinBoxSatUp">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Val low</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="horizontalSliderValLow">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBoxValLow">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Val up</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSlider" name="horizontalSliderValUp">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="spinBoxValUp">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonSelectImage">
       <property name="text">
        <string>Select image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderHueLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHueLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHueLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderHueLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderHueUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHueUp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHueUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderHueUp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSatLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSatLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSatLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderSatLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSatUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSatUp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSatUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderSatUp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderValLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxValLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>650</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxValLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderValLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderValUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxValUp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxValUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderValUp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>644</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>645</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
